backend/src/apiserver/main.go

```diff
@@ -143,7 +144,11 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 		io.WriteString(w, `{"commit_sha":"`+common.GetStringConfigWithDefault("COMMIT_SHA", "unknown")+`", "tag_name":"`+common.GetStringConfigWithDefault("TAG_NAME", "unknown")+`"}`)
 	})
 
-	topMux.Handle("/apis/", mux)
+	// log streaming is provided via HTTP.
+	runLogServer := server.NewRunLogServer(resourceManager)
+	topMux.HandleFunc("/apis/v1beta1/runs/{run_id}/nodes/{node_id}/log", runLogServer.ReadRunLog)
```

Comment:
Shall we mark the api as `v1alpha1` initially?

Reply:
@Bobgy Sure, I changed the path